cmd/kubeadm/app/constants/constants.go

```diff
@@ -46,8 +46,12 @@ const (
 	FrontProxyClientCertName           = "front-proxy-client.crt"
 	FrontProxyClientKeyName            = "front-proxy-client.key"
 
-	AdminKubeConfigFileName   = "admin.conf"
-	KubeletKubeConfigFileName = "kubelet.conf"
+	AdminKubeConfigFileName             = "admin.conf"
```

Comment:
In our experience, people will probably want a "starting point" kubeconfig that doesn't have an identity, but does have the server connection information in it.  Doesn't have to be in this pull, but it's likely to come up.